spyder/config/base.py

```diff
@@ -572,6 +572,22 @@ def running_in_mac_app(pyexec=None):
         return False
 
 
+# =============================================================================
+# Micromamba
+# =============================================================================
+def get_spy_umamba_path():

```

Comment:
Maybe in this function we could validate directly with the `running_in_mac_app()` and `is_pynsist()` functions and in the `else` we could check if `MAMBA_EXE` existis and has a valid value. What do you think @mrclary ?

Reply:
That's fine. I would not check `MAMBA_EXE` here, since I think the purpose should be limited to returning Spyder's micromamba executable, if it exists. The `find_conda` function will check `MAMBA_EXE` for a micromamba executable on the system (not in Spyder).